core/store/src/trie/resharding.rs

```diff
@@ -279,6 +331,91 @@ fn apply_delayed_receipts_to_children_states_impl(
     Ok(())
 }
 
+fn apply_promise_yield_timeouts_to_children_states_impl(
+    trie_updates: &mut HashMap<ShardUId, TrieUpdate>,
+    insert_timeouts: &[PromiseYieldTimeout],
+    delete_timeouts: &[PromiseYieldTimeout],
+    account_id_to_shard_uid: &dyn Fn(&AccountId) -> ShardUId,
+) -> Result<(), StorageError> {
+    // TODO: we can remove this check once yield execution is stabilized.
```

Comment:
Is that a "we can" or "we must"? As in once stabilized, would it break if the insert and delete timeouts happen to be empty and we return early from here? Can you track it somewhere so that we don't forget about it when stabilizing? Either the tracking issue or some TODO next to the ProtocolVersion for yield resume sounds ok.

Reply:
It is a "we can"; this behavior is correct whether or not the feature is enabled